graphql/util.go

```diff
@@ -748,6 +748,10 @@ func groupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*Gr
 	groupsMap := make(map[string][]*restModel.APIProjectRef)
 
 	for _, p := range projects {
+		if p.IsHidden() {
+			continue
+		}
+
```

Comment:
I'd suggest adding to the function comment that we ignore hidden projects, since we don't mention it in the function title